lib/puppet/provider/rabbitmq_binding/rabbitmqadmin.rb

```diff
@@ -39,7 +39,7 @@ def self.instances
     resources = []
     all_vhosts.each do |vhost|
       all_bindings(vhost).collect do |line|
-        source_name, destination_name, destination_type, routing_key, arguments = line.split(/\t/)
+        source_name, destination_name, destination_type, routing_key, arguments = line.split()
```

Comment:
Is this a necessary change, does not seem pertinent to adding the new `management_ssl` param.